django_project/sawps/models.py

```diff
@@ -18,29 +18,17 @@ class ExtendedGroup(models.Model):
     description = models.TextField()
 
     def __str__(self):
-        return self.group.name
+        return self.description
 
 
 @receiver(post_save, sender=Group)
-def create_extended_group(sender, instance, created, **kwargs):
+def save_extended_group(sender, instance, created, **kwargs):
```

Comment:
I removed the create_extended_group signal. We're going to use one signal for both the create and save events